source/_posts/2023-08-02-release-20238.markdown

```diff
@@ -536,6 +536,24 @@ superseded by three single actions `start`, `stop` and `return_to_base`.
 
 {% enddetails %}
 
+{% details "Google Nest" %}
+
+The legacy Works With Nest API is no longer supported. You must manually take
```

Comment:
Maybe just put the reference to the Nest Announcement here. At the bottom it's a bit redundant, isn't it?
```suggestion
Since the Works With [Nest API is shutting down by September 2023](https://developers.nest.com/), we removed the support for it. You must...
```

Reply:
It's not true though, this was planned first. I don't want to pass the buck to Google, this legacy unmaintained integration needs to die for the assistant specific reasons.  It was deprecated over a year ago and I had to temporarily undo it.


Maybe I'm bitter....

Reply:
Ok, didn't know that. In that case you can just leave it. 👍